src/app/components/organizacion/organizacion-create-email.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, OnInit } from '@angular/core';
+import { OrganizacionService } from './../../services/organizacion.service';
+import { ActivatedRoute } from '@angular/router';
+import { Plex } from '@andes/plex';
+
+@Component({
+    selector: 'organizacion-create-email',
+    templateUrl: 'organizacion-create-email.html'
+})
+export class OrganizacionCreateEmailComponent implements OnInit {
+
+
+    public idOrganizacion;
+    public organizacion;
+    public configuraciones: { emails: { nombre: string, email: string }[] } = {
+        emails: [
+
+            { nombre: null, email: null }
+        ]
+    };
+
+
+    constructor(
+        private route: ActivatedRoute,
+        private organizacionService: OrganizacionService,
+        public plex: Plex
+
+    ) { }
+
+    ngOnInit() {
+
+        this.route.params.subscribe(params => {
+            this.idOrganizacion = params['id'];
+            this.organizacionService.getById(this.idOrganizacion).subscribe(org => {
+                this.organizacion = org;
+
+                if (!this.organizacion.configuraciones) {
+                    this.organizacion['configuraciones'] = this.configuraciones;
+
+
+                } else {
+                    this.configuraciones = this.organizacion.configuraciones;
+
+                }
+
+            });
+        });
+    }
+
+    addEmail() {
+
+        this.organizacion.configuraciones.emails.push({ nombre: null, email: null });
+
+    }
+
+    save() {
+        // hay algun campo vacio?
+        const flag = this.organizacion.configuraciones.emails.some(e => e.nombre === null || e.email === null);
```

Comment:
Evaluar directamente la referencia
`        const flag = this.organizacion.configuraciones.emails.some(e => !e.nombre || !e.email)`